30 Day Warranty

gruetter opened the pull request:
This is the 30 Day Warranty Pattern put in practice by the PayPal folks.

Commit:
added draft pattern “30 day warranty”

Commit:
- added missing force

cewilliams commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+
+# Context
+
+Teams depends on another team accepting their contributions so that a component
```
I think this applies whether the contributing team provides the code or not. The end result that the contributing team is trying to achieve is that the feature is available to them to use, whether they, the receiving team, or another party write the code that provides the feature.

cewilliams commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+  submitted half finished contributions and subsequently filed requests for 
+  fixes that make it ready for use in production.
+- If code is contributed from outside the team, the team has the natural 
+  suspicition that the other team does not know how to write code that would
```
suspicition -> suspicion

cewilliams commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+  codebase.
+- There is the fear of the contributors not being available for support with
+  fixing bugs after the time on contribution.
+
```
Additional forces may be:
- Teams fear contributed code will lead to high(er) maintenance costs but do not know how to control for that
- Receiving teams may fear that teaching others how to contribute code will expose technical debt in their system and that visibility may be damaging 
- Receiving teams may not believe that they will get acceptable code no matter how much mentoring they provide
- Either team may not feel confident in measuring risks or certifying that they are mitigated in a contribution

cewilliams commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+- the receiving team is willing to accept contributions and able to share the
+  workload of initial adaptations/fixes
+- increased transparency and fairness
+- keeps escalations from becoming to heavyweight
```
to -> too

cewilliams commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+
+# Acknowledgement
+
+- Dirk
```
Dirk ->  Dirk-Willem van Gulik

cewilliams reviewed (changes requested):
Generally good, thanks for getting this in! Grammar changes are requested. Context and Forces changes are open for discussion.

NewMexicoKid commented on 30-day-warranty.md:
```diff
@@ -0,0 +1,75 @@
+bug fixes to the receiving team.
+
+a) clear contribution guidelines spelling out the expectations of the receiving
+   team and
```
and ...?

cewilliams committed:
Update 30-day-warranty.md

NewMexicoKid committed:
Updated and sync'd with PLoP 2017 paper.

Updated this pattern to match revised pattern in the PLoP 2017 paper on InnerSource Patterns, meeting reviewer comments.

NewMexicoKid reviewed (approved):
This has been reviewed by the PLoP 2017 author team and assigned shepherd from PLoP.
